src/raster/tests.rs

```diff
@@ -782,6 +782,12 @@ fn test_raster_histogram() {
         .map(|i| expected[i] + expected[i + 1])
         .collect::<Vec<_>>();
     assert_eq!(hist.counts(), &expected_small);
+
+    // n_buckets = 0 is not allowed
+    match rb.histogram(-0.5, 255.5, 0, true, true) {
+        Err(_) => (), // error expected
+        Ok(_) => panic!("Expected Err, got Ok")
```

Comment:
You can use unwrap_err or match against the expected error variant to make the test more robust.

Reply:
thanks, commit updated